bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartGraphs.java

```diff
@@ -29,267 +44,359 @@
+               createUI_GraphAction(container, TourManager.GRAPH_PACE);
+               createUI_GraphAction(container, TourManager.GRAPH_POWER);
+               createUI_GraphAction(container, TourManager.GRAPH_TEMPERATURE);
+               createUI_GraphAction(container, TourManager.GRAPH_GRADIENT);
+               createUI_GraphAction(container, TourManager.GRAPH_ALTIMETER);
+               createUI_GraphAction(container, TourManager.GRAPH_CADENCE);
+               createUI_GraphAction(container, TourManager.GRAPH_GEARS);
+
+               createUI_GraphAction(container, TourManager.GRAPH_RUN_DYN_STANCE_TIME);
+               createUI_GraphAction(container, TourManager.GRAPH_RUN_DYN_STANCE_TIME_BALANCED);
+               createUI_GraphAction(container, TourManager.GRAPH_RUN_DYN_STEP_LENGTH);
+               createUI_GraphAction(container, TourManager.GRAPH_RUN_DYN_VERTICAL_OSCILLATION);
+               createUI_GraphAction(container, TourManager.GRAPH_RUN_DYN_VERTICAL_RATIO);
+
+               createUI_GraphAction(container, TourManager.GRAPH_SWIM_STROKES);
+               createUI_GraphAction(container, TourManager.GRAPH_SWIM_SWOLF);
+            }
+            {
+               /*
+                * Checkbox: show in chart toolbar
+                */
+               _chkShowInChartToolbar_Altitude = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Pulse = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Speed = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Pace = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Power = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Tempterature = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Gradient = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Altimeter = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Cadence = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Gears = createUI_GraphCheckbox(container);
+
+               _chkShowInChartToolbar_RunDyn_StanceTime = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_RunDyn_StanceTimeBalance = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_RunDyn_StepLength = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_RunDyn_VerticalOscillation = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_RunDyn_VerticalRatio = createUI_GraphCheckbox(container);
+
+               _chkShowInChartToolbar_Swim_Strokes = createUI_GraphCheckbox(container);
+               _chkShowInChartToolbar_Swim_Swolf = createUI_GraphCheckbox(container);
+            }
+            {
+               /*
+                * CUSTOM TRACKS
+                */
+               _chkShowInChartToolbar_Custom_Tracks.clear();
```

Comment:
When you throw away existing UI controls, are they disposed?

Reply:
Some of the reviews could not be set to the correct line number or are only related to the file

Reply:
> When you throw away existing UI controls, are they disposed?

good catch, you are right, I forgot to dispose them before clearing them, I'll correct that.